listeners.py

```diff
@@ -50,7 +57,48 @@ def process_result(self, obj):
                                       obj.result_checksum,
                                       obj.show_reproduce_content))
 
+    def get_failed_task_name(self, task):
+        return task[0][0] + ((':' + task[0][1]) if task[0][1] else '')
+
+    def get_long_mark(self, task):
+        return '(long)' if task in self.long_tasks else ''
+
     def print_statistics(self):
+        field_size = 60
+        top_rss = 10
+        results_sorted = []
+
+        # prepare standalone subpath '<vardir>/statistics' for statistics files
+        stats_dir = os.path.join(Options().args.vardir, 'statistics')
+        safe_makedirs(stats_dir)
+
+        # RSS
+
+        # print to stdout RSS statistics for all failed tasks
+        if self.failed_tasks:
+            color_stdout('RSS of the failed tasks:\n', schema='test_var')
+            for task in self.failed_tasks:
+                task_name = self.get_failed_task_name(task)
```

Comment:
If you'll pass `task[0]` to the function, it'll be common enough to don't repeat the `task_id[0] + ((':' + task_id[1]) if task_id[1] else '')` expression. I would propose a name like `pretty_task_name()`.

Reply:
Ok, changed.